suite-native/state/src/SecureStoreProvider.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect, useState } from 'react';
+import { Provider } from 'react-redux';
+
+import { EnhancedStore } from '@reduxjs/toolkit';
+import { Persistor, persistStore } from 'redux-persist';
+
+import { StorageProvider } from '@suite-native/storage';
+
+import { initStore } from './store';
+
+type SecureStoreProviderProps = {
+    children: React.ReactNode;
+};
+
+export const StoreProvider = ({ children }: SecureStoreProviderProps) => {
+    const [store, setStore] = useState<EnhancedStore | null>(null);
+    const [storePersistor, setStorePersistor] = useState<Persistor | null>(null);
+
+    useEffect(() => {
+        async function initStoreAsync() {
```

Comment:
Please use arrow function as we use it everywhere

Reply:
bd8e22ff3